msticpy/context/azure/sentinel_analytics.py

```diff
@@ -75,7 +75,7 @@ def list_alert_rules(self) -> pd.DataFrame:
 
         """
         return self._list_items(  # type: ignore
-            item_type="alert_rules", api_version="2021-10-01"
+            item_type="alert_rules", api_version="2022-11-01-preview"
```

Comment:
Looks like this API is out of preview now - should be "2022-11-01"
https://learn.microsoft.com/en-us/rest/api/securityinsights/stable/alert-rules/list?tabs=HTTP

Reply:
I've updated